editor/src/components/navigator/navigator-utils.ts

```diff
@@ -256,7 +289,20 @@ export function getNavigatorTargets(
           }
         }
       } else {
-        fastForEach(Object.values(subTree.children), (child) => {
+        const notProcessedChildren = Object.values(subTree.children).filter(
```

Comment:
maybe something like notRenderPropChildren or normalChildren better describes that the interesting thing about these children isn't that they are unprocessed (raw) but rather the fact that we didn't deem them to be render props
```suggestion
        const notRenderPropChildren = Object.values(subTree.children).filter(
```

Reply:
https://github.com/concrete-utopia/utopia/pull/4986/commits/ec94818460bc276c593b2a5822300a55cf3aa0c2